core/rawdb/accessors_metadata.go

```diff
@@ -79,7 +79,7 @@ func ReadPreimage(db DatabaseReader, hash common.Hash) []byte {
 
 // WritePreimages writes the provided set of preimages to the database. `number` is the
 // current block number, and is used for debug messages only.
-func WritePreimages(db DatabaseWriter, number uint64, preimages map[common.Hash][]byte) {
+func WritePreimages(db DatabaseWriter, preimages map[common.Hash][]byte) {
```

Comment:
Could you please update the method doc too to get rid of the mentioned block number?

Reply:
OK, of course, no problem.

Reply:
Already updated the func doc. Could you please review? Thanks.